iosApp/iosApp/ComponentViews/RoutePill.swift

```diff
@@ -123,6 +123,8 @@ struct RoutePill: View {
                 .lineLimit(1)
                 .modifier(ColorModifier(pill: self))
                 .modifier(ClipShapeModifier(spec: spec))
+                .accessibilityElement()
+                .accessibilityLabel("\(route?.label ?? "") \(route?.type.typeText(isOnly: true) ?? "")")
```

Comment:
issue: for the GL pill which is a line instead of a route, this is read aloud as "space". where line is present instead of route, this should use line info

Reply:
By "line info" do you mean the `longName`? There also appears to be a `description()` but I can't seem to find the definition for that.

Reply:
Ah yes, longName looks appropriate for this! Looking at the [V3 API response](https://api-v3.mbta.com/lines/line-Green) for Green Line it is "Green Line", which is what we want here.

Reply:
Thanks for the answer in Slack! 0a446a03

Reply:
Nice! One last thing - I don't think we are guaranteed a line is present if a route is not since stopResult type is also an option. Seems ripe for some refactoring to make the distinct options clearer, but in the meantime I think just switching from `line!` to `line?` would be sufficient.

Reply:
Ah, thought it was either/or. Will fix

Reply:
42bcbb7b